### PR TITLE
Happy-thoughts-API Sara Stjernberg 

### PR DESCRIPTION
**Pull request by stjernberg**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -6,21 +6,73 @@ import mongoose from 'mongoose'
...
+
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.json(thoughts)
+})
+
+app.post('/thoughts', async (req, res) => {
+  
+  const thought = new Thought({ message: req.body.message  })
+
```
Comment (AnnaHellqvist) [inline server.js:60]: you could remove some white space here :)

**Review by AnnaHellqvist — APPROVED** [review]
Great job with the project Sara! 💥 Your code looks clean and readable, you could add some comments before the endpoints and explain what they do :)

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
I sent like 12 of messages from your frontend and only 1 made its way to the database. Please check it.

**Review by KarolCislowski — APPROVED** [review]
It works now perfectly 😄

**Pull request closed without merging**